src/templates/Navigator/index.js

```diff
@@ -32,6 +32,9 @@ function separateParts(children) {
 }
 
 function renderPartial(part, data) {
+  // Will not render if data is undefined
+  // To render without data, pass in true
+  if (!data) return undefined;
```

Comment:
@patrickkim haha, throwing up? This is to prevent a partial from adding empty space to the DOM when it doesn't have what it needs to render its component.